private/buf/bufgen/internal/bufgenv2/config.go

```diff
@@ -171,103 +179,105 @@ func validateExternalManagedConfigV2(externalConfig ExternalManagedConfigV2) err
 	}
 	for _, externalDisableConfig := range externalConfig.Disable {
 		if len(externalDisableConfig.FileOption) == 0 && len(externalDisableConfig.Module) == 0 && len(externalDisableConfig.Path) == 0 {
-			return errors.New("must set one of file_option, module, path for a disable rule")
+			return errors.New("must set one of file_option, module and path for a disable rule")
 		}
-		// TODO
 	}
 	for _, externalOverrideConfig := range externalConfig.Override {
-		// TODO
-		_ = externalOverrideConfig
+		if len(externalOverrideConfig.Option) == 0 {
+			return errors.New("must specify an option to override")
+		}
+		if externalOverrideConfig.Value == nil {
+			return errors.New("must specify an value to override")
+		}
 	}
 	return nil
 }
 
 func newDisabledFunc(externalConfig ExternalManagedDisableConfigV2) (disabledFunc, error) {
-	var selectorFileOption FileOption
+	if len(externalConfig.FileOption) == 0 && len(externalConfig.Module) == 0 && len(externalConfig.Path) == 0 {
```

Comment:
Everything should be `option` now, not `file_option`, unless I'm missing something...why would override and disable differ?

Reply:
I was thinking in disable a user can only disable options from `descriptor.proto`, like `java_package`,
and in override a user can override options like `java_package_prefix` and etc.

Reply:
This is the opposite of the direction we want to go in - this goes back to v1, where users have to put extra thought into what options apply where, whereas the goal of v2 is that everything just works, and you don't have to think about disable vs override as an example

Reply:
I do not want to have to read documentation to know which options apply where